pkg/options/validate.go

```diff
@@ -88,7 +88,7 @@ func (r ReleaseOptions) WriteToFile(fs billy.Filesystem, path string) error {
 	if !exists {
 		file, err = filesystem.CreateFileAndDirs(fs, path)
 	} else {
-		file, err = fs.OpenFile(path, os.O_RDWR|os.O_CREATE|os.O_TRUNC, os.ModePerm)
+		file, err = fs.OpenFile(path, os.O_RDWR|os.O_CREATE|os.O_APPEND, os.ModePerm)
```

Comment:
Hmm I'm not sure if I agree with this solution.

In the following code block that uses the file:

https://github.com/rancher/charts-build-scripts/blob/a46c0df20db94e46d54c6f4e1b673191df7d93c7/pkg/validate/validate.go#L43-L51

The behavior of the code is to modify the release.yaml to be what the user expects it to be by adding the charts that fall under one of the three categories into the release.yaml:
- `UntrackedInRelease`: charts that are locally listed that are not in the release branch
- `RemovedPostRelease`: charts that are in the release branch but not locally listed
- `ModifiedPostRelease`: charts that are in both the release branch and locally listed but have differing files

If we simply append instead of truncating, wouldn't that result in multiple runs of `make validate` adding the same chart entries multiple times? That doesn't seem right.

Reply:
I'm going to resolve this thread since this line has been reverted.